components/webext-storage/src/migration.rs

```diff
@@ -198,7 +286,30 @@ fn do_insert(tx: &Transaction<'_>, ext_id: &str, vals: Vec<(String, Value)>) ->
             ":data": &Value::Object(map),
         },
     )?;
-    Ok(())
+    Ok(num_entries)
+}
+
+#[derive(Debug, Clone, Default, PartialEq)]
+pub struct MigrationInfo {
+    /// The number of entries (rows in the original table) we attempted to
+    /// migrate. Zero if there was some error in computing this number.
```

Comment:
It would be great to clarify what each row in the original table means. Is it a single key-value pair for an extension (so this is all key-value pairs for all extensions), or something different?